internal/Picker/Picker.module.less

```diff
@@ -212,6 +246,18 @@
 				}
 			}
 
+			&.noindicator {
+				&::before {
+					.sand-spotlight-resting-bg-colors();
+				}
+
+				.incrementer,
+				.decrementer {
+					color: @sand-picker-joined-incrementer-color;
+				}
+				
+			}
+			
```

Comment:
If it has the same style with below "&.vertical" , we can jus use it
&.vertical, &.noindicator {